circus-train-s3-s3-copier/src/main/java/com/hotels/bdp/circustrain/s3s3copier/S3S3Copier.java

```diff
@@ -158,8 +159,8 @@ private void initialiseAllCopyRequests() {
         .info("Initialising all copy jobs");
     AmazonS3URI sourceBase = toAmazonS3URI(sourceBaseLocation.toUri());
     AmazonS3URI targetBase = toAmazonS3URI(replicaLocation.toUri());
-    srcClient = s3ClientFactory.newSourceInstance(sourceBase, s3s3CopierOptions);
-    targetClient = s3ClientFactory.newTargetInstance(targetBase, s3s3CopierOptions);
+    srcClient = s3ClientFactory.newInstance(sourceBase, s3s3CopierOptions);
+    targetClient = s3ClientFactory.newInstance(targetBase, s3s3CopierOptions);
```

Comment:
Would potentially like to refactor further to create a single client with source and target bases, and the role.